lib/ProductOpener/Redis.pm

```diff
@@ -0,0 +1,43 @@
+use Redis::Client;
+
+BEGIN
+{
+	use vars       qw(@ISA @EXPORT_OK %EXPORT_TAGS);
+	@EXPORT_OK = qw(
+		&push_to_search_service
```

Comment:
I think the ampersand can safely be omitted here. See https://metacpan.org/pod/Exporter#How-to-Export

> If you are only exporting function names it is recommended to omit the ampersand, as the implementation is faster this way.